test/algorithms/classifiers/test_qsvc.py

```diff
@@ -101,6 +101,19 @@ def test_qsvc_parameters(self):
 
         self.assertEqual(score, 0.5)
 
+    def test_args_warning(self):
+        """Test QSVC deprecation warning appears when *args passed in"""
+        qsvc = QSVC(1)
+
+        with self.assertWarns(DeprecationWarning):
+            print(qsvc)
```

Comment:
Can we do say `_ = str(qsvc)` in al cases so we call the string conversion that print would be doing - but unlike print it will not be printing anything to the console/log.